custom_model_runner/datarobot_drum/drum/artifact_predictors/artifact_predictor.py

```diff
@@ -16,7 +16,7 @@
     TARGET_TYPE_ARG_KEYWORD,
     TargetType,
 )
-from datarobot_drum.drum.utils import DrumUtils
+from datarobot_drum.drum.utils.drum_utils import DrumUtils
```

Comment:
DR drum drum utils drum_utils, rolls off the tongue doesn't it, hah

Reply:
:wink: 
some day this too will get refactored